test/index.test.js

```diff
@@ -81,10 +81,45 @@ describe('DID Identity Provider', function () {
     })
 
     it('DID identity with incorrect id does not verify', async () => {
-      const identity2 = { id: 'NotAnId', publicKey: identity.publicKey, signatures: identity.signatures, type: identity.type, sign: identities.sign, verify: identities.verify }
+      const identity2 = { 
+        ...identity,
+        id: 'NotAnId', 
+      }
       const verified = await identities.verifyIdentity(identity2)
       assert.strictEqual(verified, false)
     })
+
+    it('Valid DID identity with fake publicKey does not verify', async () => {
+      // HACKER part:
+      // Create a completely new set of private keys
+      const fakeSeed = new Uint8Array([100, 100, 100, 100, 100, 100, 100, 100, 100, 100, 100, 100, 100, 100, 183, 177, 123, 238, 83, 240, 143, 188, 87, 191, 33, 95, 58, 136, 46, 218, 219, 245])
+      const fakeDidProvider = new Ed25519Provider(fakeSeed)
+      const fakeProvider = await DIDIdentityProvider({ didProvider: fakeDidProvider })();
+      const fakeKeystore = await KeyStore({ path: path.join(keypath, 'fake') })
+      const fakeIdentities = await Identities({ keystore: fakeKeystore })
+
+      // Create an identity with all new keys, but with an ID from the 
+      // "admin" identity we wanna use to get write access by utilizing
```

Comment:
```suggestion
      // "admin" identity we wanna use to get write access by utilizing
```
Change to:

```
// "admin" identity we wanna use to get write access by utilizing
```

Reply:
Sorry, I'm not sure i understand what you would like to change here, the suggestion and your comment seems identical and github complains about that.
<img width="792" alt="Screenshot 2024-06-06 at 13 35 49" src="https://github.com/orbitdb/orbitdb-identity-provider-did/assets/1871220/19b16c5b-89e1-49d3-aacb-959ea69e0444">

Reply:
Sorry, the change should read:

```
"admin" identity we want to use to get write access by utilizing
```

Thanks

Reply:
Done